File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.py
def solution(s):
    answer = ''
    isOdd = False
    l = len(s)
    mid = (l//2 + 1)-1
    
    if l%2 != 0:
        answer += s[mid]
    else:
        answer += s[mid-1]
        answer += s[mid]
        # mid = l//2
        # mid2 = mid1 + 1
    return answer

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.cs
using System;
public class Solution {
    public string solution(string s) {
        string mid = "";
        int len = s.Length;
        int midIndex = len / 2;
        Console.Write(len);
        Console.Write(midIndex);
        
        if(len % 2 == 1)
            mid = s.Substring(midIndex, 1);
        else
            mid = s.Substring(midIndex-1, 2);
        return mid;
    }
}